README.md

```diff
@@ -283,13 +283,13 @@ A special type of battery device is a *FLiRS* device. These devices do not sleep
 
 ### Polling
 
-The binding supports periodic polling. This has two purposes - firstly to ensure that a device is still responding, and secondly to update the bindings representation of the state of the device.  Where possible *associations* should be used to update the device state - this will keep network traffic to a minimum, which will improve the network latency, and it will be faster since *associations* send updates to the controller immediately where polling will always be noticeably slower.
+The binding supports periodic polling. This has two purposes - firstly to ensure that a device is still responding, and secondly to update the bindings representation of the state of the device.  Where possible *associations* should be used to update the device state and polling set to days or weeks - this will keep network traffic to a minimum, which will improve the network latency, and it will be faster since *associations* send updates to the controller immediately where polling will always be noticeably slower.
```

Comment:
I'm not sure I would agree here. Polling is useful to detect if a device is DEAD - normally you want to detect that more often than every week or two. The reality is that polling even with a rate of a few hours has an extremely low impact - eg with 200 nodes on the network, and a poll rate of 4 hours, we would have network usage by polling of about 1 frame every 15 to 20 seconds, and an overall loading of around 0.02% - not so bad ;)

Reply:
Yes agree with the dead device but this should be very rare and certainly formy spare room every week check is fine. As I posted this is the reason I will leave it on for devices I rarely use.

For some i use more often I keep it at a few hours. For those I use very regularly I used to turn it off on HC2 as it was just not needed.

On the utilisation I am not convinced that your calculation is correct for all situations though as the silicon improves it may get closer to the theory. Also many devices when polled will return a few more than 1 frame but agree totally that every 4 hours should not harm a good network. I suppose it is an optimist vs  pessimist discussion we are having. It really needs to be fit for your needs but some people are a bit daft and think a kWh report every 0.1 s from every device even it not drawing power is useful. If we could get everybody to start with very little then adjust the ones that need more regular it would be a safe approach.

Reply:
You are free to do what you like and if you want to set the polling to 10 days, then this is up to you. However here you are making recommendations for the documentation, and we should be more generic and you should therefore consider what you are recommending for the masses, and not just your personal preference.

Node DO drop off the network - it is not so uncommon, and it may happen due to variable propagation during the day.

The calculation is simple - 200 devices with an average of 4 channels is 800 requests. If we assume these are spread over the 4 hours (the binding tries to avoid them all being together!), then that is 800 requests every 14400 seconds, or 1 request every 18 seconds. At 100kb/s, this is a loading that is around the 0.02%. Maybe this isn't 100% accurate, and it assumes the requests get spread over time which again isn't 100% accurate, but it's in this order of magnitude, and it shows the loading is (very!) low.

We are NOT talking about polling every 0.1s - I don't know why you keep raising this.

Reply:
4 hours would be a good start. As I say optimist vs pessimist but we appear to be the reverse on different issues. I suppose we gravitate to the issue that has bitten us most over the years. I like the sound of a stable network with 200 nodes with lots of reporting and polling every 4 hours. Have you ever managed that? If you have I know a few HC2 users that will be over in a shot.

As to the talk about reporting, Sorry as I say I am often not clear in coms. I think you are extremely precise. Some days i wish I was.

What I was talking about was combining polling with reporting as the paragraph addresses both.
 
The two basic things new users fail to understand as we see in the discussion on this board and over at Fibaro.

Far too many new users I have dealt with over the years configure polling to every minute and reporting changes every 10s for every device as they configure everything to report on a small change. They get to 30 devices with 10 sleeping nodes with an amount of routing. Then start seeing slow updates so make it even worse by configuring some even more regular polling and reporting as that seems to be the issue.

Your original text should be enough to stop them but possibly recommending 4 hours as a starter is correct and would make it better so can we agree on that?.



If we limit the change to that for now.

It would be interesting to have a further discussion on bandwidth as I still feel it saturates well below the theoretical level when I have tested. I have never put my finger on why as it seems that it should be possible to send a lot of frames as you describe. Possibly other things are going on that are not shown with a zniffer.  I know if I make users think it is less than it is. it takes a lot less time for the issue to go away That is a good result if a bit disingenuous and lacking your precision.

Anyway thanks for the invite and the challenges.  Will make the one small adjustment for now and just suggest 4hrs. It has to be better than the settings some users are trying so certainly will make things a lot better for them if they bother to read.